src/warm_up_model.py

```diff
@@ -0,0 +1,8 @@
+from sentence_transformers import SentenceTransformer
+
+from src import config
+
+if __name__ == '__main__':
+    SentenceTransformer(
```

Comment:
I assume this downloads the model ... I can't see from the config.py if we've change to stop using dropbox - re: https://climate-policy-radar.slack.com/archives/C02ET708QDP/p1686758013146269